sentry.yaml

```diff
@@ -0,0 +1,472 @@
+            valueFrom:
+               secretKeyRef:
+                 name: sentry-rds
+                 key: db.name
+          - name: SENTRY_DB_USER
+            valueFrom:
+               secretKeyRef:
+                 name: sentry-rds
+                 key: db.user
+          - name: SENTRY_DB_PASSWORD
+            valueFrom:
+               secretKeyRef:
+                 name: sentry-rds
+                 key: db.password
+          - name: SENTRY_EMAIL_HOST
+            valueFrom:
+              secretKeyRef:
+                name: smtp
+                key: server
+          - name: SENTRY_EMAIL_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: smtp
+                key: username
+          - name: SENTRY_EMAIL_USER
+            valueFrom:
+              secretKeyRef:
+                name: smtp
+                key: password
+          - name: SENTRY_EMAIL_PORT
+            valueFrom:
+              secretKeyRef:
+                name: smtp
+                key: port
+          - name: SENTRY_EMAIL_USE_TLS
+            valueFrom:
+              secretKeyRef:
+                name: smtp
+                key: require_tls
+          envFrom:
+          - configRef:
+            name: sentry
+          - secretRef:
+            name: sentry-github-oauth
+          - secretRef:
+            name: sentry-general
+          image: quay.io/app-sre/sentry:9.1.2
+          imagePullPolicy: Always
+          name: sentry-worker
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+          - mountPath: /etc/sentry/
+            name: config-volume
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        terminationGracePeriodSeconds: 30
+        volumes:
+        - configMap:
+            defaultMode: 420
+            name: sentry
+          name: config-volume
+    test: false
+    triggers:
+    - type: ConfigChange
+  status:
+    availableReplicas: 0
+    latestVersion: 0
+    observedGeneration: 0
+    replicas: 0
+    unavailableReplicas: 0
+    updatedReplicas: 0
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      run: redis
+      service: sentry
+    name: redis
+  spec:
+    ports:
+    - port: 6379
+      protocol: TCP
+      targetPort: 6379
+    selector:
+      run: redis
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+- apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      run: sentry
+      service: sentry
+    name: sentry
+  spec:
+    ports:
+    - name: http
+      port: 9000
+      protocol: TCP
+      targetPort: 9000
+    selector:
+      run: sentry-web
+    sessionAffinity: None
+    type: ClusterIP
+  status:
+    loadBalancer: {}
+parameters:
+- description: Image tag
+  displayName: Image tag
+  name: IMAGE_TAG
```

Comment:
this parameter will be passed during deployment, so it should be used in the template.